deploy/cluster_role.yaml

```diff
@@ -1163,3 +1163,19 @@ rules:
   - '*'
   verbs:
   - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: hco.kubevirt.io:config-reader
+  namespace: kubevirt-hyperconverged
```

Comment:
not exactly sure the context in which this manifest is used but the other roles do not specify the namespace.  So maybe also shouldn't here?

Reply:
I want to give access to the configmap with namespace kubevirt-hyperconverged and name kubevirt-storage-class-defaults.
if I don't restrict the namespace then I'm granting access to an identically named configmap in anu namespace (if someone happens to use that exact name. unlikely, but I shouldn't do it)